config.py

```diff
@@ -0,0 +1,6 @@
+import os
+
+CONFIGURATION = {
+        # enter the full path of downloaded bootstrap template as the value of directory_path key
```

Comment:
I would just add to this comment to explicitly state that the folio_example occupies the configuration value by default and the user should replace this default with their own path